content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,213 +114,233 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
+   value, Odoo will still allow the quantity (based on the serial numbers) to be delivered or
+   received.
 
-Once all product quantities have been assigned a serial number, click :guilabel:`Confirm` to close
-the popup, and click :guilabel:`Validate`. A :guilabel:`Traceability` smart button will appear upon
-validating the receipt. Click the :guilabel:`Traceability` smart button to see the updated
-:guilabel:`Traceability Report`, which includes: a :guilabel:`Reference` document, the
-:guilabel:`Product` being traced, the :guilabel:`Lot/Serial #`, and more.
+   .. image:: serial_numbers/stock-move-generate-quantity-tip.png
+      :align: center
+      :alt: Show how the quantity of serial numbers alters the delivery order quantity.
+- :guilabel:`Keep current lines` checkbox: Check this box to keep existing serial numbers that may
+  have been previously added. To replace existing serial numbers in the list, leave the box
+  unchecked.
 
-Manage serial numbers on delivery orders
-----------------------------------------
+After filling out these fields, click the :guilabel:`Generate` button. The newly generates serials
+will now appear in the :guilabel:`Open: Stock move` window. By clicking :guilabel:`Save`, the
+:guilabel:`Quantity` and the :guilabel:`Serial Numbers` fields on the delivery order or receipt will
+update automatically.
 
-Assigning serial numbers to **outgoing** goods can be done directly from the sales order (SO).
+Import serials
+**************
 
-To create an :abbr:`SO (sales order)`, navigate to the :menuselection:`Sales` app, and click the
-:guilabel:`Create` button. Doing so reveals a new, blank quotation form. On this blank quotation
-form, fill out the necessary information, by adding a :guilabel:`Customer`, and adding products to
-the :guilabel:`Product` lines (in the :guilabel:`Order Lines` tab), by clicking :guilabel:`Add a
-product`.
+Another option for assigning multiple serial numbers at once is to click the
+:guilabel:`Import Serials/Lots` button in the :guilabel:`Open: Stock move` pop-up window.
 
-Then, choose the desired quantity to sell by changing the number in the :guilabel:`Quantity` column.
+.. important::
+   If the import button is not visible, ensure the :guilabel:`Create New` box is checked in the
+   :ref:`receipt's configuration page <inventory/product_management/operation-type-setting>`.
 
-Once the quotation has been filled out, click the :guilabel:`Confirm` button to confirm the
-quotation. When the quotation is confirmed, the quotation becomes an :abbr:`SO (sales order)`, and a
-:guilabel:`Delivery` smart button appears.
+Doing so opens the :guilabel:`Import Serials` pop-up window. Enter each serial number on a separate
+line in the :guilabel:`Lots/Serial numbers` text field.
 
-Click the :guilabel:`Delivery` smart button to view the warehouse receipt form for that specific
-:abbr:`SO (sales order)`.
+.. image:: serial_numbers/stock-move-import-serials.png
```

Comment:
what does the **Keep current lines** option do?

Reply:
I suggest defining this here so it's not a lingering question for the reader (they have to find out what it means later in the **Example** block a few sections below this)